org.eclipse.dawnsci.remotedataset.server/src/org/eclipse/dawnsci/remotedataset/server/info/TreeServlet.java

```diff
@@ -12,17 +12,20 @@
 package org.eclipse.dawnsci.remotedataset.server.info;
 
 import java.io.IOException;
+import java.security.Provider.Service;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.security.Provider.Service'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)